app/server/middleware/apiMiddleware.ts

```diff
@@ -10,12 +10,15 @@ import {
   getCookiesOrEmptyString
 } from "./identityMiddleware";
 
-export type JsonHandler = (res: Response, jsonString: string) => void;
+export type BodyHandler = (res: Response, body: string) => void;
 
-export const apiHandler = (jsonHandler: JsonHandler) => (
+const straightThroughBodyHandler: BodyHandler = (res, jsonString) =>
+  res.send(jsonString);
+
+export const proxyApiHandler = (
   basePath: string,
   ...headersToForward: string[]
-) => (
+) => (bodyHandler: BodyHandler = straightThroughBodyHandler) => (
```

Comment:
As discussed earlier can we remove the default param so that the calls are more explicit